CONTRIBUTING.md

```diff
@@ -26,4 +26,21 @@ The simplest procedure is to run `npm run watch` which executes
 directory and will recompile any changed files. If you prefer to compile
 them manually, use `npm run compile` which will run `ucompiler go` for you.
 
+## Releasing
+
+Project members with push access to the repository also have the permissions
+needed to release a new version.  If there have been changes to the project and
+the team decides it is time for a new release, the process is to:
+
+1. Update `CHANGELOG.md` with the planned version number and a short bulleted
```

Comment:
**VERY IMPORTANT**: Add a step to run `npm run compile` to update the `lib` files from the current state of `src`. Otherwise Atom won't be running off the current state of the plugin.

(We really need to get CI deployment figured out / working...)